dls-controls/malcolmjs
# Feature/table row alarms #228

## Description

- Added alarm icons to table rows. 
- Added a "selected" row flag (which currently just highlights the row) triggered by clicking the alarm icon for that row.
- Refactored table local state to be an array of objects, each containing all the values for one row.
- Added a new icon to attributeAlarm to show a combined dirty & error state.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA:SEQ1/table
- click one of the row icons
- click into any text input for any row and type an invalid input (e.g. "dog")
- click "submit"
- see the new error icon (in the parent block details pane and at the bottom left of the middle pane)

## Agile board tracking

connect to #228 
connect to #225 
closes #228 
closes #225
